neuvector-controller-5.3.advisories.yaml

```diff
@@ -9,31 +9,16 @@ advisories:
       - GHSA-622h-h2p8-743x
     events:
       - timestamp: 2024-04-27T10:34:17Z
-        type: detection
+        type: false-positive-determination
         data:
-          type: scan/v1
-          data:
-            subpackageName: neuvector-controller-5.3
-            componentID: 484c8f3083b901da
-            componentName: github.com/neuvector/neuvector
-            componentVersion: v0.0.0-20240408172821-edbdcba63283
-            componentType: go-module
-            componentLocation: /usr/bin/controller
-            scanner: grype
+          type: vulnerable-code-version-not-used
+          note: neuvector-controller is using a non vulnerable version of neuvector module which has been released after the affected versions.
 
   - id: CVE-2024-3154
     aliases:
-      - GHSA-c5pj-mqfh-rvc3
+      - GHSA-2cgq-h8xw-2v5j
     events:
-      - timestamp: 2024-04-27T10:34:18Z
-        type: detection
```

Comment:
similar to above, we should keep the `detection` event and append a new `fixed` event after.  We don't need to remove previous events.